### PR TITLE
Zib-EpisodeOfCare

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

**Comment by jduwel** [comment]
Did a small fix, but cannot seem to find anything else. @ArdonToonstra maybe you can double check

File: resources/nl-core/nl-core-EpisodeOfCare.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="EpisodeOfCare" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-EpisodeOfCare" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="EpisodeOfCare">
+      <path value="EpisodeOfCare" />
+      <alias value="nl-core-EpisodeOfCare" />
+    </element>
```
Comment (ArdonToonstra) [inline resources/nl-core/nl-core-EpisodeOfCare.xml:30]: Missing nl-core-Patient reference. I will add it.

**Review by ArdonToonstra — COMMENTED** [review]
I am missing:
- release-notes
- EpisodeOfCare-Title (or has that now become EpisodeOfCareName?)
- EpisodeOfCare-DateFirstEncounter in the nl-core profile
- EpisodeOfCare-DateLastEncounter in the nl-core profile

**Comment by jduwel** [comment]
> * EpisodeOfCare-Title (or has that now become EpisodeOfCareName?)
> * EpisodeOfCare-DateFirstEncounter in the nl-core profile
> * EpisodeOfCare-DateLastEncounter in the nl-core profile

Why? Where can we find these concepts in some dataset that will use zib2020-r4?
And even if we already know for sure that they will be applicable to zib2020-r4, we do not have any mapping guidelines for nl-core profiles.

File: resources/zib/zib-EpisodeOfCare.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        <map value="NL-CM:16.6.4" />
+        <comment value="FocusEpisodeOfCare" />
+      </mapping>
+    </element>
+    <element id="EpisodeOfCare.period.start">
+      <path value="EpisodeOfCare.period.start" />
+      <short value="StartDate" />
+      <definition value="The date that marks the beginning of the episode of care. Usually this is the date of the first contact of the patient with the care provider in the context of the health problem." />
+      <alias value="BeginDatum" />
+      <mapping>
```
[the remark this answers is not in the document]
Comment (jduwel) [reply resources/zib/zib-EpisodeOfCare.xml]: Hmm I do not think so, that would be duplicating the spec basically. Zib descriptions make it clear enough that the episode is some period of time with a start and end.
Only if there is some sort of answer to ZIB-1434 that does not meet your analysis in #52 that we have to intervene.

File: resources/nl-core/nl-core-EpisodeOfCare.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-EpisodeOfCare.xml]: Thanks!

File: resources/zib/zib-EpisodeOfCare.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        <identity value="zib-episodeofcare-v1.0-2020EN" />
+        <map value="NL-CM:16.6.6" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="EpisodeOfCare.extension:episodeOfCareName">
+      <path value="EpisodeOfCare.extension" />
+      <sliceName value="episodeOfCareName" />
+      <max value="1" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-EpisodeOfCare.xml]: Thanks!

File: resources/zib/zib-EpisodeOfCare.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ArdonToonstra) [reply resources/zib/zib-EpisodeOfCare.xml]: You are right. Let's leave it as is.

**Comment by ArdonToonstra** [comment]
>Why? Where can we find these concepts in some dataset that will use zib2020-r4?
And even if we already know for sure that they will be applicable to zib2020-r4, we do not have any mapping guidelines for nl-core profiles.

Well, we have to put some effort into researching what those extensions are and what they are doing in the STU3 profiles. We can't just ditch everything not accounted for by the zib while we are processing the STU3 profiles. I am completely fine with not having them in the nl-core-profile but it should be a well-informed decision and stated in the release-notes.md. 
I believe these extensions are added because of the GP use case, therefore, the most logical place to inform about these extensions would be Alexander. He can give us intel on whether these extensions will be needed or not.

**Comment by jduwel** [comment]
I do not propose to ditch them, I suggest not adding them out of the blind as long as there isn't a 'zib2020-ready' dataset, which imo is a big difference between this use case and for example MP-920. If there will be a dataset at some point in the near future, we can add it then, including a clear mapping, because this is clearly lacking in STU3.  Process-wise I think this is better then copying everything that was present in STU3 without a proper mapping and I guess this is what we have been doing up until now (Patient.generalPractitioner for example)

**Comment by jduwel** [comment]
But this probably all comes down to a lack of guidelines of what we actually should and want to do with nl-core, so maybe we should first focus on determining that

**Comment by ArdonToonstra** [comment]
I agree that we should not blindly copying all that was present in STU3 and that we have somewhat lacking guidelines on what to do with the nl-core profiles. For these two extensions, I would still propose to discuss this with Alexander or GP product manager to notify them that we propose to not copy these extensions without a dataset and to state this in the release-notes.

**Comment by ArdonToonstra** [comment]
@Ashwin-nictiz , I have created the ada2fhir mapping and noticed that guidance for the mandatory `status` element is missing. Could you work on that?

**Comment by pieter-edelman-nictiz** [comment]
Handmatig gemerged.

**Pull request closed without merging**